modules/apps/portal-search/portal-search-ranking-web/test/js/components/ResultsRankingForm.es.js

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import ResultsRankingForm from 'components/ResultsRankingForm.es';
-import {cleanup, render, waitForElement} from 'react-testing-library';
+import {cleanup, fireEvent, render, waitForElement} from 'react-testing-library';
 import 'jest-dom/extend-expect';
 import '@babel/polyfill';
```

Comment:
What is `@babel/polyfill` needed for?

Reply:
It's for the async function! Has to wait for the results to load.

https://github.com/babel/babel/issues/5085